### ISSUE TITLE
speed test code available?

### ISSUE THREAD

**Issue by mla** [tokuhirom/Router-Boom#8]
Tokuhiro, I'm using your basic design and trying to add url builder and submapper support.

Do you have the code you used when comparing Router::Simple with Router::Boom? Would like to see how my changes have affected things. Just wondering how many different routes you had defined for the test, etc.

**Comment by syohex** [comment]
https://github.com/tokuhirom/Router-Boom/blob/master/author/benchmark.pl ?

**Comment by mla** [comment]
Thanks! I looked around in the distribution but missed that.

Ok, so do I win anything?

```text
                         Rate Router::Simple Router::Boom::Method Router::Boom My::Router
Router::Simple        36202/s             --                 -91%         -93%       -93%
Router::Boom::Method 405735/s          1021%                   --         -21%       -24%
Router::Boom         513912/s          1320%                  27%           --        -4%
My::Router           535914/s          1380%                  32%           4%         --
```

**Comment by mla** [comment]
Was it intentional that all of the match calls in benchmark.pl actually fail? It doesn't make that much of a difference on a relative basis, but I was surprised.

**Comment by syohex** [comment]
It is bug and fixed by #9
